test/nstemplateset/nstemplateset_assertion.go

```diff
@@ -0,0 +1,237 @@
+package nstemplateset
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// TODO: remove in favor of github.com/codeready-toolchain/toolchain-common/test/nstemplateset
+// (because there's already a `go mod replace` directive for github.com/codeready-toolchain/toolchain-common)
+
+type Assertion struct {
+	nsTmplSet      *toolchainv1alpha1.NSTemplateSet
+	client         client.Client
+	namespacedName types.NamespacedName
+	t              test.T
+}
```

Comment:
I think that this whole assertion file is already in toolchain-common repo, isn't it?

Reply:
yes, I used it here while the PR was still in review (and I was already using another version of toolchain-common)

Reply:
changed in 301bc73